common/shared.c

```diff
@@ -906,23 +906,44 @@ void init_test(struct ft_opts *opts, char *test_name, size_t test_name_len)
 	}
 }
 
+#define ft_post(post_fn, comp_fn, seq, op_str, ...)				\
```

Comment:
Can you make this upper case, to better indicate that it is a macro?  Other than that, looks good.  Thanks